library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -287,10 +292,14 @@ class Client(
     @DelicateApi("This function is delicate and should be used with caution. Adding a wallet already associated with an inboxId will cause the wallet to lose access to that inbox. See: inboxIdFromAddress(address)")
     suspend fun addAccount(newAccount: SigningKey, allowReassignInboxId: Boolean = false) {
```

Comment:
@codabrink I'm really struggling with this piece in particular. Right now we just have one interface for signers. Either we need a RootSigner and additional Signer. 🤔 That may be okay. Let me try that.